youtube_dl/extractor/ruutu.py

```diff
@@ -85,6 +86,18 @@ def _real_extract(self, url):
         formats = []
         processed_urls = []
 
+        def authenticate_video_url(url):
+            """Authenticate video url
+
+            Returns original video url with authentication token
+            appended. Required for most videos since early 2019.
+            """
+            authenticated_url = self._download_webpage(
+                'https://gatling.nelonenmedia.fi/auth/access/v2',
+                video_id, query={'stream': url},
+                note='Authenticating video url', fatal=False)
+            return authenticated_url
```

Comment:
Won't it just make sense to have it return that instead of returning the variable ?

Reply:
It does indeed. I also removed some unnecessary whitespace.